src/main/java/org/opentripplanner/graph_builder/module/ned/ElevationModule.java

```diff
@@ -66,15 +68,27 @@
     private final boolean readCachedElevations;
     private final boolean writeCachedElevations;
     private final File cachedElevationsFile;
+    private final boolean includeEllipsoidToGeoidDifference;
+    private final int geoidDifferenceCoordinateValueMultiplier;
```

Comment:
I'd like to see comments for any new fields on elevationModule (unless it's extremely obvious what they do).